mobile-app/cypress/support/onboardingCommands.ts

```diff
@@ -40,8 +41,8 @@ declare global {
 }
 
 Cypress.Commands.add('startCreateMnemonicWallet', (recoveryWords: string[]) => {
-  cy.getByTestID('create_wallet_button').click()
```

Comment:
`create_wallet_button` is not existing anymore, but there are tests still using it 
<img width="435" alt="image" src="https://user-images.githubusercontent.com/13292662/179699368-01163d7a-cc91-41e9-b2e5-b95112ae6589.png">

<img width="741" alt="image" src="https://user-images.githubusercontent.com/13292662/179699240-2f31b872-821f-4e2e-9d77-dc70c1be4d18.png">
<img width="435" alt="image" src="https://user-images.githubusercontent.com/13292662/179699368-01163d7a-cc91-41e9-b2e5-b95112ae6589.png">